content/docs/user-guide/gems/reference/environment/vegetation.md

```diff
@@ -0,0 +1,204 @@
+---
+linkTitle: Vegetation
+title: Vegetation Gem
+description: The Vegetation Gem provides tools to place natural-looking vegetation in Open 3D Engine.
+weight: 400
+toc: true
+---
+
+The **Vegetation Gem** provides tools for procedurally populating landscapes and environments. These tools comprise of system and editor components that automate the placement of static objects using a data-driven approach. You can use these tools to replace or compliment manually painting, editing, and saving every instance in your level. 
+
+Many features of the dynamic vegetation system rely on other Gems and components to supply data about the environment, such as surfaces, images, and gradient signals, to direct where and how vegetation appears. 
+
+## Dependencies
+
+- [Surface Data Gem]()
+- [Gradient Signal Gem]()
+- [FastNoise Gem]() (optional)
+
+The Surface Data Gem allows a surface, like terrain or meshes, to emit signals or tags that communicate its surface type. For example, with a Vegetation Surface Mask Filter component, you can choose which vegetation to grow on a particular surface by using an inclusion and exclusion list. You can also recapture the tags as a gradient signal by using the Surface Mask Gradient component.
+
+The Gradient Signal Gem provides components that direct data to the vegetation system, controlling the appearance of dynamic vegetation. Using gradient signals with vegetation modifiers or filters, such as Position Modifier and Distribution Filter components, produce realistic, random expressions of vegetation in the world.
+
+## Provides
+
+Source code: [`/Gems/docs/user-guide/components/reference/Vegetation/`](https://github.com/o3de/o3de/tree/development/Gems//docs/user-guide/components/reference/Vegetation/Code)
+
+API: [Vegetation Gem API Reference](https://www.o3de.org/docs/api/gems//docs/user-guide/components/reference/vegetation/index.html)
+
+System components:
+- [Vegetation Area System](/docs/user-guide/components/reference/docs/user-guide/components/reference/vegetation/vegetation-area-system-component)
+- [Vegetation Instance System](/docs/user-guide/components/reference/docs/user-guide/components/reference/vegetation/vegetation-instance-system-component)
+
+Editor components:
```

Comment:
Maybe "Entity components"? They're not just Editor components because they also exist at runtime.